### ISSUE TITLE
windeployqt is not working ?

### ISSUE THREAD

**Issue by Buanderie** [mdimura/docker-mingw-qt5#4]
Trying to use windeployqt on my succesfully generated .exe results in:
`[devel@5bd0432e0abb ~]$ x86_64-w64-mingw32-windeployqt-qt5 /home/devel/boomrtx/build/tools/boomconfig/boomconfig.exe
Unable to query qmake: qmake returns 255:`

I started digging for possible answers, but maybe you would know better :)

Nice work on that image, btw.

**Comment by mdimura** [comment]
I tried windeployqt once, but id did not work for me. Would be great if you find out how to make it work! Maybe you could ask for help in the comments to [AUR package](https://aur.archlinux.org/packages/mingw-w64-qt5-tools/).

**Comment by Buanderie** [comment]
It falls out of the scope of this issue, but the reason I needed windeployqt to work was because I linked to dynamic libraries. If there was a branch of this image with static qt libraries instead, that would offer users another solution for deployment (without too much hassle).

**Comment by mdimura** [comment]
Yep, I will split the image into several variants at some point, qt-static could be one of them. For now I copy the dlls manually in the build script as a workaround.

**Comment by Buanderie** [comment]
Just out of curiosity, from which directory do you copy the dlls ?

**Comment by mdimura** [comment]
`/usr/{x86_64-w64-mingw32,i686-w64-mingw32}/{bin,lib/qt/plugins/{imageformats,iconengines,platforms}}` for x64 and x32 builds respectively.

**Comment by vahidlazio** [comment]
I wonder, copy dlls to where? the root project directory? and we just need the exe after all to run the app or we still need the dlls?

**Comment by mdimura** [comment]
The dlls are shared libraries (as opposed to static), so yes, they are needed in addition to the exe. It is enough to copy the .dlls and .exe in the same folder to run the program.
